code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -39,9 +41,13 @@
 	return ..()
 
 /obj/item/clothing/head/helmet/space/hardsuit/attack_self(mob/user)
-	on = !on
+	if(light_broken)
+		to_chat(user, "<span class='notice'>The headlamp has been burnt out... Looks like there's no replacing it.</span>")
```

Comment:
honestly it'd be nice to be able to replace it (cable and a lightbulb with an actionbar maybe?) instead of it getting trashed permanently